templates/news-detail/news-detail.css

```diff
@@ -0,0 +1,154 @@
+/* stylelint-disable no-descending-specificity */
+
+.news-detail main .section {
+  padding: 0;
+  margin: 0;
+}
+
+.news-detail h1 {
+  text-align: center;
+  box-sizing: border-box;
+  margin: .67em 0;
+  line-height: 1.1;
+  font-size: 36px;
+  margin-top: 3px;
+  margin-bottom: 2px;
```

Comment:
You can combine these margins - 

`margin: 3px 0 2px;`

Reply:
good catch

Reply:
fixed